content/docs/1.4.0/advanced-resources/rwx-workloads.md

```diff
@@ -8,7 +8,7 @@ Longhorn supports ReadWriteMany (RWX) volumes by exposing regular Longhorn volum
 
 # Introduction
 
-For each actively in use RWX volume Longhorn will create a `share-manager-<volume-name>` Pod in the `longhorn-system` namespace. This Pod is responsible for exporting a Longhorn volume via a NFSv4 server that is running inside the Pod. There is also a service created for each RWX volume, and that is used as an endpoint for the actual NFSv4 client connection.
+Longhorn creates a dedicated `share-manager-<volume-name>` Pod within the `longhorn-system` namespace for each RWX volume that is currently in active use. The Pod facilitate the export of Longhorn volume via an internally hosted NFSv4 server. Additionally, a corresponding Service is created for each RWX volume, serving as the designated endpoint for actual NFSv4 client connections.
```

Comment:
```suggestion
Longhorn creates a dedicated share manager pod (typically named share-manager-<volume-name>) within the longhorn-system namespace of each RWX volume that is in active use. Additionally, Longhorn creates a service for each RWX volume. The share manager pod facilitates the export of Longhorn volumes via an internally hosted NFSv4 server, while the service functions as the designated endpoint for NFSv4 client connections.
```

Reply:
If the suggested changes are fine, we can copy and paste the new wording into the other sections of the document.